src/Media/MediaBody.js

```diff
@@ -0,0 +1,20 @@
+import React, {Component, PropTypes} from 'react';
+
+export default class MediaBody extends Component {
+  static propTypes = {
+    /**
+     * The heading of the media object
+     */
+    heading: PropTypes.string
+  }
+
+  render() {
+    const {heading, children, className} = this.props;
+    return (
+      <div className={`media-body ${className}`}>
+        <h4 className="media-heading">{heading}</h4>
```

Comment:
This heading should be optional. `<h4>` is also probably not generally semantically correct.